src/assets/scripts/airport/Arrival/ArrivalBase.js

```diff
@@ -301,20 +513,21 @@ export default class ArrivalBase {
 
     /**
      * Determine delay until next spawn
+     *
+     * @for ArrivalBase
+     * @method nextInterval
+     * @return {number}
      */
     nextInterval() {
-        // TODO: these next 3 vars could easily be moved to a constants file
-        // nautical miles
-        const min_entrail = 5.5;
-        // in seconds
-        const min_interval = min_entrail * (3600 / this.speed);
-        let tgt_interval = 3600 / this.frequency;
```

Comment:
@erikquinn I see this number _a lot_ throughout the app.  What is it for?  Is it 3.6 seconds (my guess) or something else?  I would like to add this to a not-yet-created constants file and make this a little bit easier to understand when reading through.